modules/cnf-configuring-the-ptp-fast-event-publisher.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ptp.adoc
+
+[id="cnf-configuring-the-ptp-fast-event-publisher_{context}"]
+= Configuring the PTP fast event notifications publisher
+
+To start using PTP fast event notifications for a network interface in your cluster, you must enable the fast event publisher in the PTP Operator `PtpOperatorConfig` custom resource (CR) and configure `ptpClockThreshold` values in a `PtpConfig` CR that you create.
+
+To configure the fast event notifications publisher, perform the following procedure.
```

Comment:
You could remove this line and similar ones in later modules.

Reply:
"To configure the fast event notifications publisher, perform the following procedure." I think I picked this habit up from a @mikemckiernan review some time ago. 

@kalexand-rh I'll remove it as you suggest, it is redundant.